README.md

```diff
@@ -764,6 +765,32 @@ $ git show :/typo
 
 *Press `q` to quit.*
 
+
+### Git Grep
+Grep pattern under your git versioned folder. 
+
+Running:
+```
+git grep aliases
+```
+will show all the files containing the string *aliases*.
+
+![git grep aliases](http://i.imgur.com/q3WkUgl.png?1)
+
+We can also use `e` and `and` flag for more advanced search.
+
+```
+ git grep -e aliases --and -e 알리아스 
```

Comment:
I think here needs explanation for what ```-e``` ( and maybe ```--and``` in case anyone doesn't understand) means first.
The first sentence in the man page for ```-e``` would be sufficient: ```The next parameter is the pattern.```